templates/zerver/help/delete-a-stream.md

```diff
@@ -7,13 +7,20 @@ However, Zulip organization administrators must use the Zulip
 **[organization administration interface](/help/change-your-organization-settings)**
 to delete streams.
 
-{!go-to-the.md!} [Delete streams](/#organization/streams-list-admin)
-{!admin.md!}
+1. From your desktop, click on the **gear**
+   (<i class="icon-vector-cog"></i>) in the upper right corner.
 
-2. Find the stream you want to delete, and click the **Delete stream** button to
+1. Select **Settings** from the dropdown.
+
+1. Click on **Organization** in the top left corner of the settings screen.
+
+1. Click on the **Delete streams** tab in the left hand menu of the
+**Organization** screen
+
+1. Find the stream you want to delete, and click the **Delete stream** button to
 the right of it.
 
-3. Upon clicking on the **Delete stream** button, a confirmation window titled
+1. Upon clicking on the **Delete stream** button, a confirmation window titled
```

Comment:
I would say this as:
"A confirmation window will appear. Click **Yes, delete this stream**."